src/GitVersionCore/Configuration/ConfigurationUtils.cs

```diff
@@ -0,0 +1,181 @@
+using System.Collections.Generic;
+using System.Linq;
+using GitVersion.Extensions;
+using GitVersion.VersioningModes;
+
+namespace GitVersion.Configuration
+{
+    public class ConfigurationUtils
+    {
+        public static void ApplyDefaultsTo(Config config)
```

Comment:
Can't this be exposed as a `Reset()` method on the `Config` class?

Reply:
renamed to `Reset`